docs/en/protocols/README.md

```diff
@@ -50,6 +50,27 @@ the following key information:
 
 The protocol is used to report events to the backend. The [doc](../concepts-and-designs/event.md) introduces the definition of an event, and [the protocol repository](https://github.com/apache/skywalking-data-collect-protocol/blob/master/event) defines gRPC services and message formats of events.
 
+
+Report `JSON` format logs via HTTP API, the endpoint is http://<oap-address>:12800/v3/events.
```

Comment:
```suggestion
Report `JSON` format events via HTTP API, the endpoint is `http://<oap-address>:12800/v3/events`.
```

Please also quote the "http://<oap-address>:12800/v3/logs" in docs/en/protocols/Log-Data-Protocol.md because `<oap-address>` cannot be rendered correctly in our website.

See this

<img width="607" alt="Screen Shot 2021-08-04 at 14 43 31" src="https://user-images.githubusercontent.com/15965696/128134262-6baadfa6-2de5-44ee-beef-41770d492ef5.png">